catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -124,6 +124,8 @@
     <reference-list id="metacardMultiActionProviders" interface="ddf.action.MultiActionProvider"
                     filter="(id=catalog.data.metacard.*)"/>
 
+    <reference-list id="injectableAttributes" interface="ddf.catalog.data.InjectableAttribute" />
```

Comment:
Shouldn't this be optional?  This might time out if there are no registered InjectableAttributes